systemSmartContractsConfig.toml

```diff
@@ -3,8 +3,8 @@
     MinStakeValue = "100000000000000000000" #100 eGLD
     UnBondPeriod = 144000
     MinStepValue = "100000000000000000000"
-    AuctionEnableNonce = 100000000000
-    StakeEnableNonce = 100000000000
+    AuctionEnableEpoch = 100000000000
```

Comment:
If epoch is uint32 then this value is overflowed and its value is unpredictable. You can use the same value used above to SwitchJailWaitingEnableEpoch = 1000000